cba/terraform-solution-implementation/06-vsphere-permissions-restrict-folders/terraform.tfvars.example

```diff
@@ -15,7 +15,7 @@ vsphere_datacenter = "sfo-w01-DC"
 
 edge_folder = "sfo-w01-fd-edge"
 local_storage_folder = "sfo-w01-fd-storage-local"
-readonly_storage_folder = "sfo-w01-fd-storage-readonly"
+readonly_storage_folder = "sfo-w01-fd-storage-readOnly"
```

Comment:
Was this intentional? Just checking.

Reply:
need to check this one, lowercase is OK as long as its the same in both places. Let me double check that as I had issues

Reply:
Fixed to lower in both places